sample.txt

```diff
@@ -0,0 +1,3 @@
+This is suneel
+This is phani
```

Comment:
Please modify phanindra kumar instead of phani

Reply:
yes modified